bench/app.py

```diff
@@ -1,55 +1,56 @@
-from __future__ import print_function
-import os
-from .utils import (exec_cmd, get_frappe, check_git_for_shallow_clone, build_assets,
-	restart_supervisor_processes, get_cmd_output, run_frappe_cmd, CommandFailedError,
-	restart_systemd_processes)
-from .config.common_site_config import get_config
-
-import logging
+import json, logging, os, re, shutil, subprocess, sys
```

Comment:
i dont think we should be going anti-pattern and doing multiple imports on single line.